Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/DefaultPrimaryPointerSelector.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Default primary pointer selector. The primary pointer is chosen among all interaction enabled ones using the following rules in order:
+    ///   1. Currently pressed pointer that has been pressed for the longest
```

Comment:
Seems like since there are known inputs and outputs for this behavior it could be good candidate for unit tests.

Reply:
It would be nice but I think we would get more value out of a play mode test. I'm quite pressed on time though so I'll leave that out of this PR if you don't mind.